src/com/intimetec/configuration/ConfigurationReader.java

```diff
@@ -0,0 +1,33 @@
+package com.intimetec.configuration;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+
+import com.intimetec.constants.Constants;
+
+public class ConfigurationReader {
+
+	public Properties getProperties(String fileName) {
+		Properties properties;
+
+		properties = new Properties();
+		try {
+			FileInputStream reader = new FileInputStream(Constants.CONFIGURATION_LOCATION + fileName);
```

Comment:
instead of using CONFIGURATION_LOCATION constant put config file in class path and user class path